code/modules/antagonists/overthrow/overthrow.dm

```diff
@@ -0,0 +1,128 @@
+#define INITIAL_CRYSTALS 5 // initial telecrystals in the boss' uplink
+
+/datum/antagonist/overthrow
+	name = "Syndicate mutineer"
+	roundend_category = "syndicate mutineers"
+	antagpanel_category = "Syndicate Mutineers"
+	job_rank = ROLE_TRAITOR
+	var/datum/team/overthrow/team
+	var/static/list/possible_useful_items
+
+/datum/antagonist/overthrow/New()
+	..()
+	if(!possible_useful_items)
+		possible_useful_items = list(/obj/item/gun/ballistic/automatic/pistol, /obj/item/storage/box/syndie_kit/throwing_weapons, /obj/item/pen/edagger, /obj/item/pen/sleepy, \
+									/obj/item/soap/syndie, /obj/item/card/id/syndicate, /obj/item/storage/box/syndie_kit/chameleon)
+
+/datum/antagonist/overthrow/on_gain()
+	create_team()
```

Comment:
You don't need to call it manually, `create_team` is called in `add_antag_datum`, you should be passing the already existing team there. Personally i feel like splitting naming and team creation is good idea here.